docs/products/postgresql/howto/prevent-full-disk.rst

```diff
@@ -0,0 +1,58 @@
+Prevent PostgreSQL full disk issues
+===================================
+
+If your Aiven for PostgreSQL service runs out of disk space, the service will start malfunctioning and precluding the proper backups creation. 
+
+To prevent this situation, Aiven automatically detects when your service is running out of free space and stops further write operations by setting the ``default_transaction_read_only``  parameter to ``ON``.
+
+With this setting in place, clients trying to execute write operations will start facing errors like::
+
+    cannot execute CREATE TABLE in a read-only transaction.
+
+To re-enable database writes you need to increase the available space, by either deleting data or upgrading to a larger plan.
+
+Increase free space by upgrading to a larger plan
+-------------------------------------------------
+
+When upgrading to a larger plan, new nodes with bigger disk space capacity are created and replace the original nodes. Once the new nodes with increased disk capacity are up and running, the disk usage returns to below the critical level and the system automatically sets the ``default_transaction_read_only`` parameter to ``OFF`` allowing write operations again.
+You can upgrade the Aiven for PostgreSQL service plan via the `Aiven console <https://console.aiven.io/>`_ or :doc:`Aiven CLI </docs/tools/cli>`. 
+
+To perform a plan upgrade via the `Aiven console <https://console.aiven.io/>`_:
+
+#. Go to the Aiven for PostgreSQL service page
+#. In the *Overview* tab scroll down to the **Service plan** section and click on **Upgrade Plan**
+#. Select the new plan with higher capacity and click **Upgrade**.
+
+Once the new nodes with increased disk capacity are up and running, the disk usage returns to below the critical level and the system automatically sets the ``default_transaction_read_only`` parameter to ``OFF`` allowing write operations again.
+
+.. Note::
+
+    If you temporarily upgrade your service to delete data, you may have to wait for the next backup to complete before you can downgrade to a smaller plan.
```

Comment:
The next section talks about enabling the deletion of data without changing plan.

It's not obvious to me why, instead, you'd upgrade size, delete, and then downgrade size, when you could instead just delete.

I think this note either needs to explain that a bit more, or else perhaps it can go away?

Reply:
I believe the baseline is, if you want to have more time to delete the data you upgrade, delete and then resize

Reply:
Will explain it better